### ISSUE TITLE
Drawer with property of scrollAreaComponent is rendering both the native scroll bar and the component

### ISSUE THREAD

**Issue by atareversei** [mantinedev/mantine#6303]
### Dependencies check up

- [X] I have verified that I use latest version of all @mantine/* packages

### What version of @mantine/* packages do you have in package.json?

7.10.0

### What package has an issue?

@mantine/core

### What framework do you use?

Next.js

### In which browsers you can reproduce the issue?

Chrome

### Describe the bug

I have added this code:

```ts
<Drawer
  offset={16}
  radius="md"
  size="80%"
  position="bottom"
  opened={isPreviewDrawerOpen}
  onClose={() => setIsPreviewDrawerOpen(false)}
  title={l.dash.cust.previewCustomer}
  scrollAreaComponent={ScrollArea.Autosize}
>
  <PreviewDrawerContent
     formValues={form.values}
   />
</Drawer>
```
and it renders two scrollbars which don't work correctly (you can't reach the end of the container by using only one of the scrollbars)

![image](https://github.com/mantinedev/mantine/assets/76823888/931f24be-9cc5-4ca6-8e3c-c0aac21884b9)
I tested it on Chrome and Firefox and it was the same.

### Self-service

- [X] I would be willing to implement a fix for this issue

**Comment by rtivital** [comment]
I cannot reproduce that on the docs website – https://mantine.dev/core/drawer/#usage-with-scrollarea
Provide a sandbox with a minimal reproduction.

**Comment by AtaReversei** [comment]
Sorry I couldn't reproduce it in a sandbox (I will try again later, I am chasing a deadline right now), but as I was inspecting the elements in the devtools I came across a div that had a `height: 100vh;` which grabbed my attention. I tried setting it to `height: 100%;` and it fixed the problem. I hope it helps.

![image](https://github.com/mantinedev/mantine/assets/76823888/32d0131c-0087-4555-9094-87b5a93b8cc2)
